eq-author-api/README.md

```diff
@@ -57,48 +66,23 @@ docker-compose build
 docker-compose up
 ```
 
-Once the containers are running you should be able to navigate to http://localhost:4000/graphiql and begin exploring the eQ Author GraphQL API.
-
 Changes to the application should hot reload via `nodemon`.
 
-### Querying pages
-
-There is no concrete `Page` type in the GraphQL schema. Instead we use a `Page` interface, which other types implement e.g. `QuestionPage` and `InterstitialPage`.
-
-To query all pages, and request different fields depending on the type, use [inline fragments](http://graphql.org/learn/queries/#inline-fragments):
-
-```gql
-query {
-  getQuestionnaire(id: 1) {
-    questionnaire {
-      sections {
-        pages {
-          id
-
-          # inline fragment for `QuestionPage` type
-          ... on QuestionPage {
-            guidance
-            answers {
-              id
-              label
-            }
-          }
-
-          # For purposes of example only. `InterstitialPage` doesn't exist yet
-          ... on InterstitialPage {
-            # doesn't exist yet
-            someField
-          }
-        }
-      }
-    }
-  }
-}
-```
+### View survey gives Error 500 (on Mac)
+
+To fix the problem, go to eq-author-api/docker-compose.yml and change the line
+
+```- PUBLISHER_URL=http://localhost:9000/publish/```
+
+to
+
+```- PUBLISHER_URL=http://docker.for.mac.localhost:9000/publish/```
+
+## Using GraphQL with Apollo web extension
 
-### Testing through GraphiQL
+GraphQL can be queried using the Apollo client for Chrome. To be able to access the API, you need to have set up the right authorization by following the steps for [authentication section in eq-author readme](../eq-author/README.md#authentication-1).
 
-There are [queries](tests/fixtures/queries.gql) and [example data](tests/fixtures/data.json) in the [fixtures folder](tests/fixtures). These can be used with graphiql to manually build up a questionnaire.
+Install the [Apollo Client Developer Tools](https://chrome.google.com/webstore/detail/apollo-client-developer-t/jdkknkkbebbapilgoeccciglkfbmbnfm) for Chrome and run it on the Author website.
```

Comment:
This isn't related to the API. Maybe some of this can be moved into the top level documentation?

Reply:
Should that whole section about "using graphql with apollo client" be moved to the top level docs then? I assumed when you ran Apollo Client in the browser, it sends the requests to the eq-author-api?

Reply:
Yeah, the API has nothing to do with apollo client.

I wonder if we also have a lot of API specific information in those top level docs as well. Maybe we should remove the docs from here and have it only at the top level? @samiwel

Reply:
Top-level docs makes sense to me

Reply:
As discussed with Sam, it was decided to remove this section as it was no need for it.